roles/logstash/templates/logstash-999-output-es-bro.j2

```diff
@@ -7,11 +7,7 @@ output {
         }
 
         elasticsearch {
-          {% if groups['elasticsearch'] | length > 1 %}
-      	  hosts => [{% for host in groups['es_data'] %}"{{ host }}"{% if not loop.last %},{% endif %}{% endfor %}]
-          {% else %}
-      	  hosts => ["127.0.0.1:9200"]
-          {% endif %}
+          hosts => [{% for host in groups['es_data'] %}"{{ host }}"{% if not loop.last %},{% endif %}{% endfor %}]
```

Comment:
Just need to account for all in one where we're listening on loopback. Same for all the outputs.

Reply:
I did test an all-in-one install with this logic and it worked.  The only thing to note is that I am running a little differently since I only use the playbooks on a system that is already stood up.